sdk/eventhub/event-hubs/CONTRIBUTING.md

```diff
@@ -57,5 +57,9 @@ npm run build
 
 If you want to run or debug tests in this project, please see our [Test README](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/testhub/README.md).
 
+## Developer Contributing Guide
```

Comment:
If there is duplication between this file and the root contributing.md I'd suggest we remove the duplication and just refer people to the root contributing.md.

Reply:
There are some extra steps that we wanted to highlight for folks wanting to contribute to Event Hubs. How do you suggest we do that?

Reply:
I'm fine with having extra steps that are unique to event hubs here I just want to avoid duplication across these files.

Reply:
I think the steps here are specific to event-hubs so I don't think it's a duplication @weshaggard
That is why I added a link to the Readme for eventhubs to point to this so it's like
 [Eventhubs readme] => [eventhubs contribute.md] => [root contribute.md]

Reply:
A lot of the stuff in this file seems to be pretty general contributing guidelines which we should cover in our root contributing.md, if we aren't already.

Reply:
I think the part about Contribution License Agreement is not mentioned under the root contributing.md . Should we move this part there instead?

Reply:
@ramya-rao-a  Also I think except for these sections - "Run/Debug Tests", "Building the library" and "AMQP Dependencies" the rest all looks like it's applicable to all the libraries, should I move those sections under the root contributing.md instead?

Reply:
Yes the CLA stuff should definitely be moved up. For the other things I think we should probably generalize those and move them up as well.

Reply:
I have updated the files and the root contributing.md